web/src/app/administration/classification/details/classification-details.component.ts

```diff
@@ -110,10 +111,9 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
         }
         this.fillClassificationInformation(this.selectedClassification ? this.selectedClassification : new ClassificationDefinition())
       }
-
-      if (id && id !== '') {
-        this.selectClassification(id);
-      }
+        if (this.classification ? this.classification.classificationId !== id : true && id && id !== '') {
```

Comment:
Same as above
if(id && id !== '' && this.classification && this.classification.classificationId !== id){ this.selectClassification(id)}

Also extract this.classification ? this.classification.classificationId === classificationSelected.classificationId into a function isSameClassificationId(id) and use it in both if()